daemon/config/config.go

```diff
@@ -58,6 +58,12 @@ const (
 	LinuxV1RuntimeName = "io.containerd.runtime.v1.linux"
 	// LinuxV2RuntimeName is the runtime used to specify the containerd v2 runc shim
 	LinuxV2RuntimeName = "io.containerd.runc.v2"
+
+	// SeccompProfileDefault is the built-in default seccomp profile.
+	SeccompProfileDefault = "default"
```

Comment:
I think this naming choice is probably a little bit ambiguous -- if I set the daemon-wide `--secomp-profile` to something other than the built-in profile, does `--security-opt seccomp=default` get me the daemon-wide default, or the Docker built-in profile?  (Maybe `builtin` or `docker` are better name options if it's the latter?)